src/engine/qcommon/common.cpp

```diff
@@ -52,25 +52,27 @@ Maryland 20850 USA.
 #include "sys/sys_events.h"
 #include <common/FileSystem.h>
 
+Cvar::Cvar<bool> com_developer = Cvar::Cvar<bool>( "developer", "", Cvar::TEMPORARY, false );
+Cvar::Cvar<int> com_speeds = Cvar::Cvar<int>( "com_speeds", "", Cvar::NONE, 0 );
+Cvar::Cvar<float> com_timescale( "timescale", "", Cvar::CHEAT | Cvar::SYSTEMINFO, 1.0f );
+
 cvar_t *com_pid; // bani - process id
 
-cvar_t *com_speeds;
-cvar_t *com_developer;
-cvar_t *com_timescale;
-cvar_t *com_dropsim; // 0.0 to 1.0, simulated packet drops
+Cvar::Range<Cvar::Cvar<float>> com_dropsim = Cvar::Range<Cvar::Cvar<float>>( "com_dropsim", "simulated packet drop", Cvar::CHEAT, 0.0f, 0.0f, 1.0f );
 
 Cvar::Cvar<bool> cvar_demo_timedemo(
     "demo.timedemo",
     "Whether to show timing statistics at the end of a demo",
     Cvar::CHEAT | Cvar::TEMPORARY,
     false
 );
-cvar_t *com_sv_running;
-cvar_t *com_cl_running;
-cvar_t *com_version;
 
-cvar_t *com_unfocused;
-cvar_t *com_minimized;
+cvar_t *com_version;
+Cvar::Cvar<bool> com_sv_running = Cvar::Cvar<bool>( "sv_running", "", Cvar::ROM, false );
+Cvar::Cvar<bool> com_unfocused = Cvar::Cvar<bool>( "com_unfocused", "", Cvar::ROM, false );
+Cvar::Cvar<bool> com_minimized = Cvar::Cvar<bool>( "com_minimized", "", Cvar::ROM, false );
+//NOTE: this one is never used in that file, this may belong somewhere else
+Cvar::Cvar<bool> com_cl_running = Cvar::Cvar<bool>( "cl_running", "", Cvar::ROM, false );
```

Comment:
I think you can remove the ` = Cvar::Cvar<bool>` part of these lines

Reply:
nitpick.